client/lib/cgutil/cpuset_manager_v1.go

```diff
@@ -119,24 +134,25 @@ func (c *cpusetManager) CgroupPathFor(allocID, task string) CgroupPathGetter {
 
 }
 
+// task name -> task cgroup info
 type allocTaskCgroupInfo map[string]*TaskCgroupInfo
 
 // Init checks that the cgroup parent and expected child cgroups have been created
 // If the cgroup parent is set to /nomad then this will ensure that the /nomad/shared
 // cgroup is initialized.
-func (c *cpusetManager) Init() error {
-	cgroupParentPath, err := getCgroupPathHelper("cpuset", c.cgroupParent)
+func (c *cpusetManagerV1) Init(_ []uint16) error {
```

Comment:
No need to remember the initial reserve-able cores in the v1 manager, because they're handled implicitly through the cgroup hierarchy.